hooks/conan-center.py

```diff
@@ -708,7 +708,15 @@ def test(out):
 
     @run_test("KB-H050", output)
     def test(out):
-        if conanfile.name in ["opencl-icd-loader", "paho-mqtt-c", "tbb", "pdal", "vulkan-loader"]:
+        ALLOWLIST = (
+            "onetbb",
+            "opencl-icd-loader",
+            "paho-mqtt-c",
+            "pdal",
+            "tbb",
+            "vulkan-loader",
+        )
+        if conanfile.name in ALLOWLIST:
```

Comment:
```suggestion
        if conanfile.name in allowlist:
```

Lower-case for local variables.